docs/src/data/mockServices.tsx

```diff
@@ -0,0 +1,89 @@
+import { Amplify} from 'aws-amplify';
+
+const awsExports = {
```

Comment:
I'd add a comment above this that this is a fake awsexports that is needed to remove auth config errors, for future readers.

Reply:
+1

Reply:
Done